generators/client-2/templates/src/main/webapp/app/account/_account.module.ts

```diff
@@ -18,7 +18,13 @@ import {
     PasswordComponent,
     PasswordResetInitComponent,
     PasswordResetFinishComponent,
-    SettingsComponent
+    SettingsComponent,
+<% if (enableSocialSignIn) { %>
```

Comment:
this is gonna cause dangling comma when the option is not selected. @flaviencathala can you please review and make sure all such things are taken care of. Plz let me know when its ready